packages/preview/fauve-cdb/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "fauve-cdb"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = [
+    "Timothé Albouy <timothe.albouy@gmail.com>",
+    "Augustin Godinot <https://grodino.github.io>",
+]
+license = "MIT-0"
+description = "The unofficial implementation of the Collège Doctoral de Bretagne thesis manuscript template."
+categories = ["thesis"]
+disciplines = ["computer-science", "mathematics"]
+
+[template]
+path = "example"
```

Comment:
It's rather unconventional to use `template` for the package sources and `example` for what would usually be called `template`. Though it's ok if you prefer it like th is.

Reply:
Sorry for the delayed answer, I'll change this then.